packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -287,7 +289,8 @@ function AccountMenu({
           },
           { name: 'export', text: 'Export' },
           { name: 'reconcile', text: 'Reconcile' },
-          syncEnabled &&
+          syncServerStatus === 'online' &&
+            syncEnabled &&
```

Comment:
IMO the user should be able to unlink even if the server isn’t working — does this make sense?

Reply:
Good idea https://github.com/actualbudget/actual/pull/742/commits/695f04715b9f6e8eb059cf1dbcf9c74d2617b73f